src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public class Task {
+    protected String task;
+    protected Boolean isDone;
+    public static int totalTask = 0; //total number of tasks saved
+
+    //Constructor
+    public Task() {
+        this.task = "";
+        this.isDone = false;
+    }
+
+    public Task(String task){
+        this.task = task;
+        this.isDone = false;
+    }
+
+    //getters and setters
+    public String getTask() {
```

Comment:
Perhaps standardise whether there is a space in between the () and {} brackets?
```suggestion
    public String getTask(){
```

Reply:
Oh, thank you for telling me this issue! I forgot to use CTRL + ALT + L after finished writing the code haha!